test/e2e/crcsuite/crcsuite.go

```diff
@@ -266,9 +262,13 @@ func CheckOutputMatchWithRetry(retryCount int, retryTime string, command string,
 	return matchErr
 }
 
-// CheckCRCStatus checks that output of status command
-// matches given regex
 func CheckCRCStatus(state string) error {
+	if state == "running" {
+		// crc start can finish succesfully, even when
```

Comment:
double `s` in `successfully` -- I wouldn't see this if GH actions didn't complain :)